batchflow/models/torch/base.py

```diff
@@ -982,7 +1037,8 @@ def _train(self, *args, fetches=None, sync_frequency=True, sam_rho=0.0, sam_indi
         inputs = inputs[0] if isinstance(inputs, (tuple, list)) and len(inputs) == 1 else inputs
 
         # Apply model, compute loss and gradients
-        predictions = self.model(inputs)
+        with torch.cuda.amp.autocast(enabled=self.amp):
```

Comment:
what if there is no cuda installed?

Reply:
On `cpu` version of `PyTorch`, a warning of the `cuda is not installed so not using its features` sort is raised